app/js/renderer.js

```diff
@@ -83,6 +83,7 @@ function updateBody(data, settings, scripts) {
   }
   $('#content').html(generatedHtml);
 
+  refreshIframe();
```

Comment:
Yep. I just think the method should accommodate other potential workarounds rather than saying explicit what the code is doing. Perhaps even something like `forceLoadDynamicObjects` or something.